libclamav/libclamav.map

```diff
@@ -61,6 +61,15 @@ CLAMAV_PUBLIC {
     cl_engine_stats_enable;
     lsig_sub_matched;
 };
+CLAMAV_0.104.0 {
+  global:
+    cl_engine_set_clcb_engine_compile_progress;
+    cl_engine_set_clcb_sigload_progress;
```

Comment:
I just realized we should add `cl_engine_set_clcb_engine_free_progress` here.  It seems to be missing entirely from libclamav.map.